src/pe_reports/report_generator.py

```diff
@@ -55,6 +62,19 @@ def load_template():
     return prs
 
 
+def load_customers():
+    """Export PowerPoint report set to output directory."""
+    try:
+        names_obj = {}
```

Comment:
We typically prefer this syntax for clarity.

```suggestion
        names_obj = dict()
```

Reply:
There is an LGTM alert with this addition. Both as `{} `or `dict()`
We can suppress with `lgtm [py/multiple-definition]` at end of line 68. Would this be acceptable for dev-standards.

Reply:
The alert has nothing to do with the syntax.  The problem is that `names_obj` is defined here as well as on line 71.  Just delete line 68 and use my suggestion below.

Reply:
Doh - that makes sense - suggestion withdrawn!